src/android/LocalNotification.java

```diff
@@ -561,6 +669,23 @@ private void success(CallbackContext command, boolean arg) {
         command.sendPluginResult(result);
     }
 
+    /**
+     * Invoke error callback with a string boolean argument.
+     *
+     */
+    private void sendScheduleResult(CallbackContext command,
```

Comment:
Instead of having two `command.sendPluginResult()` calls, maybe we could do something like 

`command.sendPluginResult(error != null ? new PluginResult(status, error.toJSONObject()) : new PluginResult(status))`

_Observation:_  If `new PluginResult(status, null)` is possible and is the same as `new PluginResult(status)`, we could also go with that approach.

Reply:
`new PluginResult(status, null)` is not possible.